kotlinx-coroutines-core/common/src/flow/Builders.kt

```diff
@@ -90,7 +90,6 @@ public fun <T> (suspend () -> T).asFlow(): Flow<T> = unsafeFlow {
 /**
  * Creates a flow that produces values from the given iterable.
  */
-@ExperimentalCoroutinesApi
 public fun <T> Iterable<T>.asFlow(): Flow<T> = unsafeFlow {
```

Comment:
I makes sense to move `unsafeFlow` builder to a separate file, so that all the builders have can be written as `flow { ... }` for the source-browsing pleasure.